### ISSUE TITLE
Copy to explain a bit more on the Funders/Recipients pages

### ISSUE THREAD

**Issue by mariongalley** [ThreeSixtyGiving/grantnav#827]
We need some additional copy on the Funders/Recipients page to:

- [x] Explain Org IDs
   - This top section would benefit from some information about this e.g. "This organisation has been referred to using several organisation identifiers in the 360Giving data. This may be because they have both a charity number and company number."
It would be great if we could link to some explanation either in the standard docs or the data quality tool about use of Org IDs
- [x] Display Other names below the Org ID info
   - Add information about Other names e.g. "This organisation has been referred to using several names in the 360Giving data. This may be because their name can be abbreviated or written in various ways, or because it has changed over time, or because one charity number is shared by several bodies receiving distinct grants."
- [x] Explain the Grant search & turn it into a button
    - Say "See all Grants [funded/received]" instead of "Grants search"

**Comment by KDuerden** [comment]
Is 'Other names' from 360 data or is it taken from FindthatCharity sources?

**Comment by michaelwood** [comment]
Reference:

http://grantnav.360dev1.vs.mythic-beasts.com/funders
grantnav/frontend/templates/search_funders.html

http://grantnav.360dev1.vs.mythic-beasts.com/recipients
grantnav/frontend/templates/search_recipients.html

I also think we added a bit of copy on the entries we could lose "Amount" on each of the labels as that is a bit redundant and creates large amount of text density on each entry (it doesn't appear on Total Grants for instance).

**Comment by mariongalley** [comment]
On the list view pages, I've reviewed and am happy with the changes. I've ticked off the two that are done.

I think we need a tooltip similar to "Learn more about Orgs" that says "Learn more about Org IDs" to satisfy the "Explain Org IDs" item, however there isn't currently any copy in the GrantNav helpsite to explain this.

@KDuerden is happy to add this Org IDs explanation to the GrantNav helpsite. Could you add a tooltip to the List pages next to the Org IDs? @michaelwood

**Comment by michaelwood** [comment]
> Explain Org IDs

We have added the multiple org-id explainer for the summary box e.g. http://grantnav.360dev1.vs.mythic-beasts.com/org/360G-ArcadiaFund

> Could you add a tooltip to the List pages next to the Org IDs? @michaelwood

Are the list pages: http://grantnav.360dev1.vs.mythic-beasts.com/recipients and http://grantnav.360dev1.vs.mythic-beasts.com/funders ?

I think we may be blocked on that until there is an explanation added to the grantnav help site that we can link to?

**Comment by mariongalley** [comment]
> Are the list pages: http://grantnav.360dev1.vs.mythic-beasts.com/recipients and http://grantnav.360dev1.vs.mythic-beasts.com/funders ?

Yep, that's right @michaelwood 

> I think we may be blocked on that until there is an explanation added to the grantnav help site that we can link to?

That's fine, @KDuerden and I will work on this.

**Comment by michaelwood** [comment]
For now shall I copy the org-ids help text from insights? and add that as a basic tooltip on the text "Org IDs:"?
e.g.

![image](https://user-images.githubusercontent.com/493697/198018108-65b8c1b1-a404-4f99-8d20-c9d7dbc9fd57.png)

> 360Giving data identifies recipient and funding organisations in two main ways - by name and with a unique organisation identifier. These org IDs often use an official reference, such as a charity or company number or school number.

**Comment by mariongalley** [comment]
@michaelwood There's already a Learn more about Org IDs tooltip on the individual org pages, which currently links out to the GrantNav helpsite:

![image](https://user-images.githubusercontent.com/82958988/198080456-7be22da6-db34-4ac5-a18c-4f650c8a9a40.png)

However, one of my suggestions for that page is that instead of having it as a yellow i button it gets its own larger button similar to Show Additional Data - as I don't think people would expect a tooltip to take them to a new page.

So for this list view, I think it does make sense to have it as a yellow i button but for it to behave like an actual tooltip i.e. open a model rather than direct you to a new page. Happy for you to use the proposed placeholder copy from Insights in that case.

**Comment by michaelwood** [comment]
I'd advise keeping the "Learn more about organisations" on the org page as-is as that helps reassure/reinforce the design pattern of those being help items.

Just to help us describe things - those yellow box things are  "[alert tags](https://cdn.threesixtygiving.org/components/detail/alert-tag--default.html)", and they can also optionally have tooltips (i.e. text that appears on hover)

I agree it would be better on the org search results page not to repeat it, I'll have a look at what it looks like just at the top, I think that should be fine.

**Comment by michaelwood** [comment]
Updated design mock up-

![image](https://user-images.githubusercontent.com/493697/198255488-714bf5b1-c2fb-4abc-a3bb-fddbbd4efd46.png)

**Comment by michaelwood** [comment]
@mariongalley to write full copy for the teal-box

**Comment by mariongalley** [comment]
@michaelwood Draft copy:
>**Where is this data from?**
>This data was published in the 360Giving data standard by one or more publishers, with additional data including organisation identifiers and names added via FindThatCharity.

We're doing a fully copy 'sweep' and will share a doc with finalised copy, but for testing and layout purposes the above will do.

**Comment by mariongalley** [comment]
This is done. Will raise a new ticket for the full copy update as it's not part of the initial scope of this ticket and makes it harder to see what happened later.
